src/ViewModels/CkanGroupViewModel.js

```diff
@@ -159,40 +168,57 @@ CkanGroupViewModel.prototype._load = function() {
+        };
+
+    var handleError = function() {
+            throw new ViewModelError({
+                sender: that,
+                title: 'Group is not available',
+                message: '\
+    An error occurred while invoking package_search on the CKAN server.  \
+    <p>If you entered the link manually, please verify that the link is correct.</p>\
+    <p>This error may also indicate that the server does not support <a href="http://enable-cors.org/" target="_blank">CORS</a>.  If this is your \
+    server, verify that CORS is enabled and enable it if it is not.  If you do not control the server, \
+    please contact the administrator of the server and ask them to enable CORS.  Or, contact the National \
+    Map team by emailing <a href="mailto:nationalmap@lists.nicta.com.au">nationalmap@lists.nicta.com.au</a> \
+    and ask us to add this server to the list of non-CORS-supporting servers that may be proxied by \
+    National Map itself.</p>\
+    <p>If you did not enter this link manually, this error may indicate that the group you opened is temporarily unavailable or there is a \
+    problem with your internet connection.  Try opening the group again, and if the problem persists, please report it by \
+    sending an email to <a href="mailto:nationalmap@lists.nicta.com.au">nationalmap@lists.nicta.com.au</a>.</p>'
             });
-        } else {
-            populateGroupFromResults(that, json);
-        }
-    }).otherwise(function() {
-        throw new ViewModelError({
-            sender: that,
-            title: 'Group is not available',
-            message: '\
-An error occurred while invoking package_search on the CKAN server.  \
-<p>If you entered the link manually, please verify that the link is correct.</p>\
-<p>This error may also indicate that the server does not support <a href="http://enable-cors.org/" target="_blank">CORS</a>.  If this is your \
-server, verify that CORS is enabled and enable it if it is not.  If you do not control the server, \
-please contact the administrator of the server and ask them to enable CORS.  Or, contact the National \
-Map team by emailing <a href="mailto:nationalmap@lists.nicta.com.au">nationalmap@lists.nicta.com.au</a> \
-and ask us to add this server to the list of non-CORS-supporting servers that may be proxied by \
-National Map itself.</p>\
-<p>If you did not enter this link manually, this error may indicate that the group you opened is temporarily unavailable or there is a \
-problem with your internet connection.  Try opening the group again, and if the problem persists, please report it by \
-sending an email to <a href="mailto:nationalmap@lists.nicta.com.au">nationalmap@lists.nicta.com.au</a>.</p>'
+        };
+
+    //FYI: to filter by group the filterQuery param should be set to something like 
+    //  'q=groups%3dSurface%20Water&fq=res_format%3aWMS' in init_xx.json
+    var promises = [];
```

Comment:
This note should go on the documentation for the `filterQuery` property so that people building apps on NM can see it without studying our source code.